Resources/public/css/styles.css

```diff
@@ -345,6 +345,28 @@ td.sonata-ba-list-field .editable-empty:not(.editable-open) {
     width: auto; /* See https://github.com/sonata-project/SonataAdminBundle/issues/2950 */
 }
 
+/* icheck */
+.checkbox div[class^="icheckbox"],
```

Comment:
Because when I activate iCheck on config.yml the checkbox is placed "outsided" of the form.

I can try to make before / after screenshots if you want.